server/src/main/java/com/objectcomputing/checkins/services/workingenvironment/WorkingEnvironmentController.java

```diff
@@ -0,0 +1,104 @@
+package com.objectcomputing.checkins.services.workingenvironment;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.netty.channel.EventLoopGroup;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.inject.Named;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Scheduler;
+import reactor.core.scheduler.Schedulers;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.UUID;
+import java.util.concurrent.ExecutorService;
+
+@Controller("/services/working-environment")
+@Secured(SecurityRule.IS_AUTHENTICATED)
```

Comment:
Replace line 25 with:

`@Secured(RoleType.Constants.HR_ROLE)`

... and import:

`import com.objectcomputing.checkins.services.role.RoleType;`